components/ArticlesView.tsx

```diff
@@ -89,17 +91,21 @@ const ArticlesView: React.ElementType<ArticlesViewProps> = ({
         marginRight: -SECTION_PADDING,
       }}
     >
-      {initPosts.concat(extraPosts).map(post => (
-        <EachArticleView key={post.id} displayExcerpt={displayExcerpt}>
-          <TextOnlyArticle
-            post={post}
-            displayCategory={displayCategory}
-            displayExcerpt={displayExcerpt}
-            displayDateAuthor={displayDateAuthor}
-            textColor={textColor}
-          />
-        </EachArticleView>
-      ))}
+      {initPosts.concat(extraPosts).map(
+        post =>
+          ((!noCartoons && post.tsdPrimaryCategory.name !== "Cartoons") ||
```

Comment:
@hs4man21 you can simplify this (so it's more readable) to:

`noCartoons || (post.tsdPrimaryCategory.name !== "Cartoons")`